### ISSUE TITLE
Add a language comparison page

### ISSUE THREAD

**Issue by sonnyp** [sonnyp/Workbench#696]
Similar to the "Extensions" window, we should have a "Languages" window with a comparison table and links.

Similar to https://developer.gnome.org/documentation/introduction/languages.html but with a bit more focus on Workbench and developer experience

This will have 2 objectives

1. Help newcomers decide which language to use in Workbench
2. Encourage language maintainers to implement features 

Here is an example

|            | Formatter | Linter | Library demos | Support | Recommended for |
|------------|-----------|--------|---------------|---------|-----------------|
| JavaScript | Yes       | Yes    | 100           | 100%    | Beginners       |
| Vala       | No        | Yes    | 90            | 80%     | Intermediate    |
| Rust       | Yes       | No     | 90            | 80%     | Intermediate    |
| Python     | No        | No     | 0             | 50%     | Beginners       |

Support would refer to UX, APIs and features, for example

* https://github.com/sonnyp/Workbench/issues/110
* https://github.com/sonnyp/Workbench/issues/125
* https://github.com/sonnyp/Workbench/issues/120
* https://github.com/sonnyp/Workbench/issues/630
* workbench.build 

Note that these aren't Vala specific anymore.

**Comment by bragefuglseth** [comment]
Yeah! I think this should be in the already existing Extensions window, though, below the list of installed extensions. Seems like this is the place where it would be most relevant.

**Comment by gregorni** [comment]
I feel like there should also be links to the language API documentation.
